tracer.go

```diff
@@ -71,8 +75,15 @@ const sqlOperationUnknkown = "UNKNOWN"
 
 // sqlOperationName attempts to get the first 'word' from a given SQL query, which usually
 // is the operation name (e.g. 'SELECT').
-func sqlOperationName(query string) string {
-	parts := strings.Fields(query)
+func sqlOperationName(stmt string, fn func(string) string) string {
```

Comment:
nit: may I suggest turning this into a method of `Tracer`?

Reply:
Good idea!

Reply:
Seems like a good idea. Do you want that in this PR or an other one?

Reply:
Turned the function into a method 🪄